Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ttf-parser"
-version = "0.24.1"
+version = "0.25.0"
```

Comment:
Please don't change the version number in PRs. Just the other changes will do and we can set the version as appropriate during the release process.